h/search/logging_filters.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""Logging Filters."""
```

Comment:
The code in this module isn't actually specific to search at all, so I would suggest we put it in the `h.util` package instead, so that it'll be easier to find if we have the same problem in another context.